pages/robot/tabs/components/index.php

```diff
@@ -284,6 +563,254 @@ function render_components(data) {
+                        try {
+                            const response_obj = JSON.parse(result.message);
+
+                            if (response_obj.type == "object") {
+                                $('#' + output_id).html(json_to_html(response_obj));
+                            } else if (response_obj.type == "stream") {
+                                $('#' + output_id).html(json_to_html(response_obj));
+
+                                // setup live stream of data
+                                try {
+                                    let stream_topic = extract_stream_topic_from_json(response_obj);
+
+                                    let update_id = output_id + "-stream"
+                                    let update_div = $('<div>').attr('id', update_id);
+                                    $('#' + output_id).append(update_div);
+                                    stream_data(
+                                        ros,
+                                        robot_name,
+                                        stream_topic.test_topic_name,
+                                        stream_topic.test_topic_type,
+                                        update_id,
+                                        modal_id,
+                                    );
+                                } catch (error) {
+                                    console.log("Stream type response received, but an error has occurred. Error msg:" + error);
+                                }
+                            }
+                        } catch (error) {
+                            $('#' + output_id).html("Error parsing the response: " + result.message);
+                        }
+                    });
+                })
+
+                $.ajax({
+                    url: window.location.href,
+                    type: "POST",
+                    data: {id_str_read: id_str_name},
+                    success: function(response) {
+                        let special_str = "___" + id_str_name + "___";
+                        let entry = response.split(special_str)[1];
+                        // console.log("Found record: " + entry);
+                        if (entry !== "") {
+                            console.log(`[${id_str_name}] Found verification record: ${entry}`);
+                        }
+                        update_style_based_on_records(id_str_name, entry);
+                    }
+                });
+
+                // user confirms success
+                $('#' + btn_id_success).click(function() {
+                    let text = "Do you confirm the test was successful?";
+                    if (confirm(text) == true) {
+                        console.log("Success");
+                        // console.log(window.location.href)
+                        $.ajax({
+                            url: window.location.href,
+                            // url: '<?php echo $base_url; ?>/components/process.php',
```

Comment:
remove?

Reply:
The ajax call is to be removed, after the test result records are stored in the Dashboard DB. I removed the commented unused lines.